modules/sections.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable class-methods-use-this */
+export default class Sections {
+  constructor() {
+    return null;
+  }
+
+  sections() {
+    const listLink = document.getElementById('list-link');
+    const addLink = document.getElementById('add-link');
+    const contactLink = document.getElementById('contact-link');
+
+    const listSec = document.getElementById('list');
+    const addSec = document.getElementById('add-book');
```

Comment:
- [ ] [OPTIONAL] it would be better if you use es6 `arrow function` instead of `normal method` in order to practice working with both